wordle-web/src/views/game.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+    <header>
+        <h1> Pretty Much Just Wordle </h1>
+    </header>
+    <body>
+        You wouldn't steal a car.
+    </body>
```

Comment:
This content should be in a `<v-card>` component as per specifications.

Reply:
I missed the word 'card' in the requirements, should be fixed now.